cmake/FindPostgreSQL.cmake

```diff
@@ -31,6 +31,7 @@ else(POSTGRESQL_INCLUDE_DIR AND POSTGRESQL_LIBRARIES)
 
     find_path(POSTGRESQL_INCLUDE_DIR libpq-fe.h
         ${T_POSTGRESQL_INCLUDE_DIR}
+		/usr/pgsql-*/include # needed if installation is performed based on instructions here: https://www.postgresql.org/download/linux/redhat/
```

Comment:
You can remove the comments, (keeping code clean) the comment on the PR anyway

Reply:
Done